package.json

```diff
@@ -14,38 +14,38 @@
   "version": "0.1.0",
```

Comment:
NPM vs YARN, it's important NOT to mix those on the project. Both are package managers. It's good to write in the READMe which one we will be using. I don't have any preference between yarn or NPM. These days both are good.